pom.xml

```diff
@@ -67,12 +67,12 @@
     <version.javax.inject>1</version.javax.inject>
     <version.javax.annotation-api>1.3.2</version.javax.annotation-api>
 
-    <version.org.infinispan>10.0.0.Beta5</version.org.infinispan>
-    <version.org.infinispan.protostream>4.3.0.Alpha9</version.org.infinispan.protostream>
+    <version.org.infinispan>10.0.0.CR1</version.org.infinispan>
+    <version.org.infinispan.protostream>4.3.0.Alpha10</version.org.infinispan.protostream>
 
-    <version.io.quarkus>0.23.2</version.io.quarkus>
+    <version.io.quarkus>0.27.0</version.io.quarkus>
     <version.com.oracle.substratevm>19.2.0.1</version.com.oracle.substratevm>
```

Comment:
@cristianonicolai @mswiderski 0.27.0 requires GraalVM 19.2.1
https://quarkus.io/blog/quarkus-0-27-0-released/

Reply:
@ricardozanini I looked into that and apparently it takes a while for the Maven artifacts to be published. See https://github.com/quarkusio/quarkus/blob/master/bom/runtime/pom.xml#L77

Reply:
though quarkus recommends to use 19.2.1 as part of 0.27 release https://quarkus.io/blog/quarkus-0-27-0-released/

Reply:
@mswiderski correct, its just that we can't update the pom.xml until the artifacts are uploaded. That's the same issue that Quarkus itself has.